mmv1/products/workstations/WorkstationCluster.yaml

```
@@ -229,3 +229,12 @@ properties:
           output: true
           item_type:
             type: KeyValuePairs
+  - name: 'tags'
+    type: KeyValuePairs
```

if the field is immutable, you can mark that here:

```suggestion
    type: KeyValuePairs
    immutable: true
```